### DISCUSSION TITLE
Bad image generation results

### DISCUSSION THREAD

**Discussion by Charnell** [lllyasviel/stable-diffusion-webui-forge#1305] [Q&A]
Welcome,
Please tell me why Forge does not generate images according to what Lora contains. The result of generating an image is drastically different every time and does not resemble the one in the link below despite the same input parameters. Image from my Forge UI:

https://civitai.com/images/23667655

My image:
![00007-3144868156](https://github.com/user-attachments/assets/60b9c149-a446-47d2-88ea-0dcecb4227f0)

Generating data:

score_9, score_8_up, score_7_up,
1girl, alleri4, blue eyes, pointy ears, <lora:AlleriaWindrunnerTWW-10:1>, full body shot,
Negative prompt: score_1, score_2, score_3, source_comic, source_furry, NEGATIVE_HANDS, worst quality, low quality, text, censored, deformed, bad hand, monochrome, logo
Steps: 20, Sampler: DPM++ 2M Karras, CFG scale: 7, Seed: 3144868156, Size: 512x712, Model hash: 67ab2fd8ec, Model: ponyDiffusionV6XL_v6StartWithThisOne, Clip skip: 2, Version: f0.0.17v1.8.0rc-latest-277-g0af28699

**Comment by Piton1986** [comment]
> Добро пожаловать, Пожалуйста, скажите мне, почему Forge не генерирует изображения в соответствии с тем, что содержит Lora. Результат генерации изображения каждый раз кардинально отличается и не похож на тот, что показан по ссылке ниже, несмотря на одинаковые входные параметры. Изображение из моего пользовательского интерфейса Forge.:
> 
> https://civitai.com/images/23667655
> 
> Мое изображение: ![00007-3144868156](https://private-user-images.githubusercontent.com/4280613/359212213-60b9c149-a446-47d2-88ea-0dcecb4227f0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQwOTY5OTAsIm5iZiI6MTcyNDA5NjY5MCwicGF0aCI6Ii80MjgwNjEzLzM1OTIxMjIxMy02MGI5YzE0OS1hNDQ2LTQ3ZDItODhlYS0wZGNlY2I0MjI3ZjAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgxOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MTlUMTk0NDUwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YjQwNjExOWI2YWE1OGY1ZTExNzExZjRkNjg5MzcxNjBkODQwMjVmOGIwMmJhZmI4YjkzMmJhYjEwZjEyOGJiYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.Au7iu_nhsuXarLB7o3Ph5xzfn2kEsLDDHOk6PQyjmC0)
> 
> Генерация данных:
> 
> score_9, score_8_up, score_7_up, 1girl, alleri4, blue eyes, pointy ears, lora:AlleriaWindrunnerTWW-10:1, full body shot, Negative prompt: score_1, score_2, score_3, source_comic, source_furry, NEGATIVE_HANDS, worst quality, low quality, text, censored, deformed, bad hand, monochrome, logo Steps: 20, Sampler: DPM++ 2M Karras, CFG scale: 7, Seed: 3144868156, Size: 512x712, Model hash: 67ab2fd8ec, Model: ponyDiffusionV6XL_v6StartWithThisOne, Clip skip: 2, Version: f0.0.17v1.8.0rc-latest-277-g0af28699

Hi. First of all, the data is a little different

Steps: 30, Sampler: Euler a, CFG scale: 7, Seed: 3144868156, Size: 768x1344, Model hash: 821aa5537f, Model: autismmixSDXL_autismmixPony, VAE hash: 235745af8d, VAE: sdxl.vae.safetensors, Denoising strength: 0.4, Clip skip: 2, ADetailer model: face_yolov8n.pt, ADetailer confidence: 0.3, ADetailer mask only top k largest: 1, ADetailer dilate erode: 4, ADetailer mask blur: 4, ADetailer denoising strength: 0.4, ADetailer inpaint only masked: True, ADetailer inpaint padding: 32, ADetailer version: 24.4.2, Hires upscale: 1.2, Hires steps: 12, Hires upscaler: R-ESRGAN 4x+, Lora hashes: "AlleriaWindrunnerTWW-10: a0c6969596dd", Version: f0.0.17v1.8.0rc-latest-277-g0af28699

**Comment by Liva24** [comment]
I think it might be the resolution, your are using a pony/sdxl model so try something higher like 896 x 1152
